phpunit.xml.dist

```diff
@@ -17,6 +17,9 @@
         <filter>
             <whitelist>
                 <directory>./lib/Underscore/</directory>
+                <exclude>
+                    <directory>./tests/Underscore/Test/Fixture</directory>
```

Comment:
Adding this mostly to keep code coverage from being affected by test-only classes.